src/app/layouts/stepper/stepper-overview-sample/stepper-overview-sample.component.ts

```diff
@@ -0,0 +1,102 @@
+import { ChangeDetectorRef, Component, ViewChild } from '@angular/core';
+import { IgxStepperComponent } from 'igniteui-angular';
+
+@Component({
+    selector: 'app-stepper-overview-sample',
+    styleUrls: ['./stepper-overview-sample.component.scss'],
+    templateUrl: './stepper-overview-sample.component.html'
+})
+export class StepperOverviewSampleComponent {
+    @ViewChild('stepper', { static: true }) public stepper: IgxStepperComponent;
+
+    public today = new Date();
+
+    public cards = [
+        {
+            img: 'assets/images/stepper/card-gold.png',
+            price: 400,
+            offer: 'STATEMENT CREDIT OFFER',
+            type: 'Business Customized Advanced',
+            description: 'Cash Mastercard'
+        },
+        {
+            img: 'assets/images/stepper/card-red.png',
+            price: 600,
+            offer: 'STATEMENT CREDIT OFFER',
+            type: 'Business Travel Advanced',
+            description: 'World Mastercard'
+        },
+        {
+            img: 'assets/images/stepper/card-blue.png',
+            price: 500,
+            offer: 'STATEMENT CREDIT OFFER',
+            type: 'Business Golden',
+            description: 'World Mastercard'
+        }
+    ];
+
+    public states = [
+        'Alabama', 'Arizona', 'California', 'Colorado', 'Florida',
+        'Georgia', 'Hawaii', 'Illinois', 'Louisiana', 'Minnesota',
+        'Nevada', 'New York', 'New Jersey', 'Ohio', 'Texas', 'Virginia', 'Washington'
+    ];
+
+    public selectedCard;
+
+    public businessInformation = {
+        name: '',
+        physicalAddress: '',
+        city: '',
+        state: '',
+        zip: null,
+        taxIdNumber: null,
+        differentAddress: false,
+        nonUSBusinessActivity: null
+    };
+
+    public personalInformation = {
+        firstName: '',
+        lastName: '',
+        jobTitle: '',
+        phoneNumber: '',
+        email: '',
+        authorization: false,
+        agreementAccepted: false
+    };
+
+    public shippingDetails = {
+        firstName: '',
+        lastName: '',
+        mailingAddress: '',
+        city: '',
+        state: '',
+        zip: null
+    };
+
+    constructor(private cdr: ChangeDetectorRef) {
+
+    }
+
+    public selectCard(card) {
```

Comment:
We should consider adding explicit types to methods' parameters, return values, class fields and variables. And do that for all samples.